container/Makefile

```diff
@@ -0,0 +1,53 @@
+POD_NAME = faf-pod
+VOLUME_NAME = faf-db-volume
+DB_PASS="NotSoSecretPassword"
+
+build:
+	podman build -t faf-image -f Dockerfile ../
+	podman tag faf-image abrt/faf-image
+
+build_local:
+	podman build -t faf-image-local -f Dockerfile_local ../
+
+build_db:
+	podman build -t postgres-semver -f Dockerfile_db ../
+	podman tag postgres-semver abrt/postgres-semver
+
+run:
+	podman run --pod $(POD_NAME) --name faf -dit -e PGHOST=localhost -e PGUSER=faf -e PGPASSWORD=scrt -e PGPORT=5432 -e PGDATABASE=faf -e RDSBROKER=redis://faf-redis:6379/0 -e RDSBACKEND=redis://faf-redis:6379/0 abrt/faf-image
+
+run_local:
+	podman run --pod $(POD_NAME) --name faf -dit -e PGHOST=localhost -e PGUSER=faf -e PGPASSWORD=scrt -e PGPORT=5432 -e PGDATABASE=faf -e RDSBROKER=redis://faf-redis:6379/0 -e RDSBACKEND=redis://faf-redis:6379/0 faf-image-local
+
+run_db:
+	podman volume create $(VOLUME_NAME) 2>/dev/null
```

Comment:
Add `||:`?

Right now it will fail if I alread have a volume created (and I don't want to delete it).

Reply:
Done.

Reply:
Oh, I meant to add a `-` in front of the command but the outcome of `||:` indeed is more elegant.